tests/MongoDB.Driver.Examples/ChangeStreamExamples.cs

```diff
@@ -161,15 +161,10 @@ public void ChangestreamExample4()
                         "{ $addFields : { newField : 'this is an added field!' } }");
 
                 var collection = database.GetCollection<BsonDocument>("inventory");
-                using (var changeStream = collection.Watch(pipeline))
+                using (var cursor = collection.Watch(pipeline))
```

Comment:
This example appears to be missing from https://docs.mongodb.com/manual/changeStreams/#resume-a-change-stream, so we may need a docs ticket if this change doesn't automatically get picked up when we merge this PR.

Reply:
Yep, I will keep track of documentation examples after the merge.